libs/core/src/lib/dialog/dialog.component.ts

```diff
@@ -136,6 +140,15 @@ export class DialogComponent extends DialogBase implements OnInit, OnChanges, Af
         super.ngOnInit();
         this._listenOnHidden();
         this.buildComponentCssClass();
+
+        
+        if (this._rtlService) {
+            this._subscriptions.add(
+                this._rtlService.rtl.subscribe((rtl) => {
+                    this._renderer.setAttribute(this._elementRef.nativeElement, 'dir', rtl ? 'rtl' : 'ltr');
```

Comment:
If we going to reuse the rtl-ltr property, I would suggest adding second parameter for the callback which reflects current direction an enum